ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh

```diff
@@ -107,7 +107,7 @@ oc adm ocp-certificates regenerate-leaf -n openshift-config-managed secrets kube
 
 # distribute trust across all known clients
 # update our local CA bundle so that when new serving certs are used for kube-apiserver we will trust them
-oc config refresh-ca-bundle
+oc config refresh-ca-bundle || true
```

Comment:
Try 
```
oc config refresh-ca-bundle || true
sleep 2
oc config refresh-ca-bundle
```

Reply:
done

Reply:
rollback the change according to Ke's comment